adrian/index.js

```diff
@@ -0,0 +1,13 @@
+//1
+// NaN stands for Not a Number.
+
+// "hello" is not NaN;
+console.log(Number.isNaN('hello'));
+
```

Comment:
You forgot to check if 3 is NaN or not!
console.log(isNaN(3));

Reply:
Ups, right. Done! Danke!

Reply:
Passiert:) Gerne:)